cwl/src/test/scala/wdl4s/cwl/ExportCwlSamplesSpec.scala

```diff
@@ -1,12 +1,23 @@
 package wdl4s.cwl
 
-import io.circe.Printer
+import io.circe.yaml.Printer
 import org.scalatest.{FlatSpec, Matchers}
-import shapeless.Coproduct
+import shapeless.{Coproduct, Witness}
+import wdl4s.cwl.CommandLineTool.{BaseCommand, Outputs}
+import wdl4s.cwl.CwlVersion.CwlVersion
+import wdl4s.cwl.WorkflowStep.{Inputs, Run}
 
 class ExportCwlSamplesSpec extends FlatSpec with Matchers {
 
-  it should "export 1st tool" in {
+  val yamlPrinter = Printer.spaces2.copy(dropNullKeys = true, preserveOrder = true)
+
+  def assertCorrectJson(cwl: Cwl, expectedYaml: String): Unit = {
+    val cwlJson = CwlCodecs.encodeCwl(cwl)
+    val cwlYaml = yamlPrinter.pretty(cwlJson)
```

Comment:
if you add this call as a function in  `CwlCodecs` tthen everyone will have X -> Yaml and they will be happy about that.

i.e. `encodeWorkflow(workflow) : Yaml`  might be preferable to returning Json.  It could be provided in addition to the Json function